draft-ietf-ppm-dap.md

```diff
@@ -1295,7 +1296,13 @@ An aggregation job can be thought of as having three phases:
 - Completion: Finish the aggregate flow, yielding an output share corresponding
```

Comment:
"aggregate flow" or "aggregation flow"?  "aggregation flow" or "aggregation job"?

Reply:
Changed to "aggregation flow" for consistency with above. We also refer to this as the "aggregation sub-protocol".